music-player/src/components/PlaylistsComponent.vue

```diff
@@ -66,8 +116,11 @@ export default defineComponent({
       tempPlaylistsData: {
         name: '',
         SongsList: [],
```

Comment:
Please review my PR with models update. Feel free to use those changes here :)

Reply:
If already applied, can We remove my PR?

Reply:
yes, i remove your branch